pkg/eosclient/eosgrpc/eosgrpc.go

```diff
@@ -650,7 +666,7 @@ func (c *Client) GetFileInfoByPath(ctx context.Context, auth eosclient.Authoriza
 	log.Info().Str("func", "GetFileInfoByPath").Str("uid,gid", auth.Role.UID+","+auth.Role.GID).Str("path", path).Msg("")
 
 	// Initialize the common fields of the MDReq
-	mdrq, err := c.initMDRequest(ctx, auth)
+	mdrq, err := c.initMDRequest(context.Background(), auth)
```

Comment:
I think would be better if here we still create an empty context but with the (eventual) cancel signal from the `ctx` passed as parameters.
In this way, if the caller cancel the context (for a timeout, CTRL-C, etc...), the grpc call to EOS will be cancelled.

I'm not aware of something similar in the standard library, so we should define it in reva (in `ctxpkg`).

Reply:
Hi gmgigi,

 I don't really know the details of what you propose. Would you like to sit together and have a look at this?